src/main/java/ca/corefacility/bioinformatics/irida/ria/utilities/FileUtilities.java

```diff
@@ -236,20 +234,44 @@ private static String formatName(String name) {
 	}
 
 	/**
-	 * Get file extension from filename.
+	 * Get file extension from filepath.
 	 * <p>
 	 * Uses simple regex to parse file extension {@code ^.*\.(\w+)$}.
 	 *
-	 * @param filename Filename
+	 * @param filepath The {@link Path} of a file to retrieve ext.
 	 * @return File extension if found; otherwise empty string
 	 */
-	public static String getFileExt(String filename) {
-		Matcher matcher = regexExt.matcher(filename);
+	public static String getFileExt(Path filepath) {
+		Matcher matcher = regexExt.matcher(filepath.getFileName().toString());
 		String ext = "";
 		if (matcher.matches()) {
-			ext = matcher.group(1);
+			ext = matcher.group(1).toLowerCase();
 		}
-		return ext.toLowerCase();
+		if (ext.equals("html")) {
```

Comment:
Do you think you could maybe add another condition in here `ext.equals("html.zip")`? That way we can also display the `quast.html.zip` output from previous run results of the **Assembly Annotation Collection** pipeline (though there are multiple `html` files in the zip archive in this case and only one will be chosen, though I think that's alright for now).

Maybe going too far outside of the scope of what we want to accomplish though, since we mainly just want to support a single zipped HTML file.

Reply:
Awesome. It works. Thanks so much:

![image](https://user-images.githubusercontent.com/200517/149548161-cf1b5986-e354-4a4d-85c8-b18ad6e76343.png)